### PR TITLE
Adding Veda's recomendation for Clarksburg

### PR DESCRIPTION
**Pull request by vedashreevinay**
- [x] Add commit 
- [x] Review comment 
- [x] Official review
- [ ] merge

### CODE REVIEW

**Comment by jml9904** [comment]
OK, here's a comment and one of these little pictures. 🍕

File: vedashreevinay-clarksburg.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Clarksburg, MD 
+
+### Places to Eat 
+- Dunkin donuts
+- REd Robin
+- Carrabas
+
```
Comment (jml9904) [inline vedashreevinay-clarksburg.md]: Or drive into Georgetown :)

Comment (vedashreevinay) [reply vedashreevinay-clarksburg.md]: Haha! Georgetown is awesome :)

**Review by jml9904 — APPROVED** [review]
Nice work!

**Pull request merged**